### DISCUSSION TITLE
cloudwatch plugin ec2 need disaplay instance tag name instead of instance id.

### DISCUSSION THREAD

**Discussion by Seven2Nine** [grafana/grafana#56612] [Data Sources]
![image](https://user-images.githubusercontent.com/61679388/194832250-780447b2-2ba9-4240-9759-3e6f66d8a1f6.png)
If just can watch instance id will make people confuse.

**Comment by sarahzinger** [comment]
Hi there,
As part of work moving from [discussions to issues](https://github.com/grafana/grafana/discussions/73424), I'm going to close this discussion, in favor of [this issue](https://github.com/grafana/grafana/issues/77953). 
Thanks for the suggestion!
